src/test/java/io/github/perrymant/moneymaker/MoneyFormatterTest.java

```diff
@@ -0,0 +1,18 @@
+package io.github.perrymant.moneymaker;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MoneyFormatterTest {
+
+    @Test
+    public void canIntToCurrencyFormat() {
```

Comment:
What about:
- [ ] 0
- [ ] negative values
- [ ] decimals smaller than 1 pound

Add separate test cases for these.